hosting-companies.md

```diff
@@ -66,18 +66,17 @@ In alphabetical order:
 * [Servebolt.com](https://servebolt.com)
 * [Simplenet](https://simplenet.io)
 * [Simply.com](https://www.simply.com)
-* [Site5](http://www.site5.com/)
+* [Site5](https://www.site5.com/)
 * [SiteDistrict](https://sitedistrict.com/)
-* [SiteGround](http://www.siteground.com/)
+* [SiteGround](https://www.siteground.com/)
 * [SpinupWP](https://spinupwp.com/)
 * [The Email Shop](https://theemailshop.co.uk/)
 * [Templ](https://templ.io/)
 * [True-False Hosting](https://truefalsehosting.com)
 * [TrulyWP](https://trulywp.com/)
-* [TVC.Net](http://TVC.Net/)
+* [TVC.Net](https://tvcnet.com/)
 * [UNLIMITED.RS](https://unlimited.rs/)
 * [Veerotech](https://www.veerotech.net/)
-* [Viral Hosting](https://viralhosting.dk)
```

Comment:
I have removed this entry as this site just lists the sites. It is not a hosting service anymore.